collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -139,17 +139,10 @@ public void onCreate(Bundle savedInstanceState) {
         ((SupportMapFragment) getSupportFragmentManager().findFragmentById(R.id.map)).getMapAsync(googleMap -> {
             setupMap(googleMap);
             locationClient.setListener(this);
+            locationClient.start();
         });
     }
 
-    @Override
-    protected void onStart() {
-        super.onStart();
-        Collect.getInstance().getActivityLogger().logOnStart(this);
```

Comment:
I think it's fine to remove this line because we are removing the ActivityLogger, but it'd be good to get this separate commit next time.